### PR TITLE
PWGCF: fixed QA flags

### PR DESCRIPTION
**Pull request by abilandz**

### CODE REVIEW

File: PWGCF/MultiparticleCorrelations/Tasks/multiparticle-correlations-ab.cxx [hunk 1]
```diff
@@ -216,7 +216,7 @@ struct MultiparticleCorrelationsAB // this name is used in lower-case format to
   // -------------------------------------------
 
   // F) Process only converted simulated Run 2 data:
-  void processSim_Run2(CollisionSim const& /*collision*/) // TBI 20240517 extend this subscription eventually
+  void processSim_Run2(CollisionSim const& collision) // TBI 20240517 extend this subscription eventually
```
Comment (victor-gonzalez) [inline PWGCF/MultiparticleCorrelations/Tasks/multiparticle-correlations-ab.cxx:219]: Same here

File: PWGCF/MultiparticleCorrelations/Tasks/multiparticle-correlations-ab.cxx [hunk 2]
```diff
@@ -234,7 +234,7 @@ struct MultiparticleCorrelationsAB // this name is used in lower-case format to
   // -------------------------------------------
 
   // H) Process both converted reconstructed and corresponding MC truth simulated Run 1 data;
-  void processRecSim_Run1(CollisionRecSim_Run1 const& /*collision*/, aod::BCs const&, TracksRecSim const& /*tracks*/, aod::McParticles const&, aod::McCollisions const&)
+  void processRecSim_Run1(CollisionRecSim_Run1 const& collision, aod::BCs const&, TracksRecSim const& tracks, aod::McParticles const&, aod::McCollisions const&)
```
Comment (victor-gonzalez) [inline PWGCF/MultiparticleCorrelations/Tasks/multiparticle-correlations-ab.cxx:237]: Same here

File: PWGCF/MultiparticleCorrelations/Core/MuPa-MemberFunctions.h [hunk 3]
```diff
@@ -2080,7 +2105,7 @@ void BookParticleCutsHistograms()
   if (pc.fUseParticleCuts[ePtDependentDCAxyParameterization]) {
     pc.fPtDependentDCAxyFormula = new TFormula("fPtDependentDCAxyFormula", pc.fsParticleCuts[ePtDependentDCAxyParameterization].Data());
     // As a quick insanity check, try immediately to evaluate something from this formula:
-    if (std::isnan(pc.fPtDependentDCAxyFormula->Eval(1.44))) {
+    if (isnan(pc.fPtDependentDCAxyFormula->Eval(1.44))) {
```
Comment (victor-gonzalez) [inline PWGCF/MultiparticleCorrelations/Core/MuPa-MemberFunctions.h:2108]: Any reason for removing the `std::` on isnan?
I think this was introduced directly on gitHub to standardize the code and perhaps you are still using an old local version. Please, check it

File: PWGCF/MultiparticleCorrelations/Core/MuPa-MemberFunctions.h [hunk 4]
```diff
@@ -4303,10 +4339,10 @@ bool ValidTrack(T const& track)
   // c) Additional validity checks for all tracks (in Run 3, 2 and 1), use only during debugging:
   if (tc.fInsanityCheckForEachParticle) {
 
-    // *) std::isnan() check (remember that 'nan' is 0./0., inf-inf, etc. However 'inf' itself is NOT a 'nan', therefore std::isnan(1./0.) is false, std::isnan(0./0.) is true, etc.):
-    if (std::isnan(track.phi()) || std::isnan(track.pt()) || std::isnan(track.eta())) {
+    // *) isnan() check (remember that 'nan' is 0./0., inf-inf, etc. However 'inf' itself is NOT a 'nan', therefore isnan(1./0.) is false, isnan(0./0.) is true, etc.):
+    if (isnan(track.phi()) || isnan(track.pt()) || isnan(track.eta())) {
```
Comment (victor-gonzalez) [inline PWGCF/MultiparticleCorrelations/Core/MuPa-MemberFunctions.h:4343]: Same here

File: PWGCF/MultiparticleCorrelations/Tasks/multiparticle-correlations-ab.cxx [hunk 5]
```diff
@@ -189,7 +189,7 @@ struct MultiparticleCorrelationsAB // this name is used in lower-case format to
   // -------------------------------------------
 
   // C) Process only simulated data:
-  void processSim(CollisionSim const& /*collision*/, aod::BCs const&, TracksSim const& /*tracks*/)
+  void processSim(CollisionSim const& collision, aod::BCs const&, TracksSim const& tracks)
```
[the remark this answers is not in the document]
Comment (abilandz) [reply PWGCF/MultiparticleCorrelations/Tasks/multiparticle-correlations-ab.cxx]: This was rolled back by mistake - I pushed a new commit, which brings this back!

File: PWGCF/MultiparticleCorrelations/Core/MuPa-MemberFunctions.h (hunk 3 above)
Comment (abilandz) [reply PWGCF/MultiparticleCorrelations/Core/MuPa-MemberFunctions.h]: I didn't pull these changes in my class done directly on GitHub, that's the only reason. In the latest commit, I switched back to std::

**Review by victor-gonzalez — APPROVED** [review]

**Comment by victor-gonzalez** [comment]
Thanks!

**Pull request merged**